src/rpc/common/Validators.hpp

```diff
@@ -427,71 +427,73 @@ class CustomValidator final {
 [[nodiscard]] bool
 checkIsU32Numeric(std::string_view sv);
 
-/**
- * @brief Provides a commonly used validator for ledger index.
- *
- * LedgerIndex must be a string or an int. If the specified LedgerIndex is a string, its value must be either
- * "validated" or a valid integer value represented as a string.
- */
-extern CustomValidator LedgerIndexValidator;
+struct CustomValidators final {
```

Comment:
Please add doxygen documentation for this struct.

Reply:
Sure. Just added in the new commit